ods_ci/tests/Resources/CLI/DataSciencePipelines/DataSciencePipelinesBackend.resource

```diff
@@ -0,0 +1,115 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with Data Science Pipelines via CLI
+Library          OperatingSystem
+Resource         ../../../Resources/OCP.resource
+
+
+*** Variables ***
+${DSPA_PATH}=      tests/Resources/Files/pipeline-samples/v2/dspa
+
+
+*** Keywords ***
+Create Pipeline Server
+    [Documentation]    Creates a pipeline server providing object storage and database information
+    ...    Note: currently, only some of the parameters are used. In the future this keyword will be
+    ...    enhanced to use them all
+    [Arguments]    ${namespace}
+    ...    ${object_storage_access_key}    ${object_storage_secret_key}
+#    ...    ${object_storage_endpoint}      ${object_storage_region}
+#    ...    ${object_storage_bucket_name}
+#    ...    ${database_host}=${EMPTY}    ${database_port}=3306
+#    ...    ${database_username}=${EMPTY}    ${database_password}=${EMPTY}
+#    ...    ${database_db_name}=${EMPTY}
+    ...    ${dsp_version}=v2
+
+    Create Secret With Pipelines Object Storage Information    namespace=${namespace}
+    ...    object_storage_access_key=${object_storage_access_key}
+    ...    object_storage_secret_key=${object_storage_secret_key}
+
+    # Process DSPA Template to create pipeline server
+    ${TEMPLATE_PARAMETERS}=    Set Variable    -p DSP_VERSION=${dsp_version}
+    Run      oc process -f ${DSPA_PATH}/dspa-template.yaml ${TEMPLATE_PARAMETERS} | oc apply -n ${namespace} -f -
+
+# robocop: disable:line-too-long
+Create PipelineServer Using Custom DSPA
```

Comment:
Can it be a parameter of `Create Pipeline Server` above?

Reply:
The "Create Pipeline Server" is using an OpenShift Template to create the pipeline server. I prefer to have two different keywords in this case